src/test/groovy/org/jetbrains/intellij/ExternalPluginRepositorySpec.groovy

```diff
@@ -24,12 +24,12 @@ class ExternalPluginRepositorySpec extends IntelliJPluginSpecBase {
 
         then:
         assert plugin != null
-        assert plugin.file.name == 'org.intellij.plugins.markdown-master-8.5.0.20160208'
+        assert plugin.artifact.name == 'markdown'
         assert collectFilePaths(plugin.jarFiles, repository.cacheDirectory.absolutePath) ==
-                ['/plugins.jetbrains.com/org.intellij.plugins.markdown-master-8.5.0.20160208/lib/markdown-javafx-preview.jar',
-                 '/plugins.jetbrains.com/org.intellij.plugins.markdown-master-8.5.0.20160208/lib/Loboevolution.jar',
-                 '/plugins.jetbrains.com/org.intellij.plugins.markdown-master-8.5.0.20160208/lib/intellij-markdown.jar',
-                 '/plugins.jetbrains.com/org.intellij.plugins.markdown-master-8.5.0.20160208/lib/markdown.jar'] as Set
+                ['/plugins.jetbrains.com/org.intellij.plugins.markdown-master-8.5.0.20160208/markdown/lib/markdown-javafx-preview.jar',
+                 '/plugins.jetbrains.com/org.intellij.plugins.markdown-master-8.5.0.20160208/markdown/lib/Loboevolution.jar',
+                 '/plugins.jetbrains.com/org.intellij.plugins.markdown-master-8.5.0.20160208/markdown/lib/intellij-markdown.jar',
+                 '/plugins.jetbrains.com/org.intellij.plugins.markdown-master-8.5.0.20160208/markdown/lib/markdown.jar'] as Set
```

Comment:
I believe original download name may be required in some cases (markdown plugin seems to be looking for its css files by path markown/lib/default.css) so I have changed layout of the repository little bit. It is /plugins.jetbrains.com/**pluginId**-**pluginVersion**/**unzipped_downloaded_artifact** now